api/api.go

```diff
@@ -101,3 +101,70 @@ func OrbQuery(ctx context.Context, logger *logger.Logger, configPath string) (*C
 			}
 		}`)
 }
+
+type CreateNamespaceResponse struct {
+	Namespace struct {
+		CreatedAt string
+		Id string
+	}
+
+	Errors []struct {
+		Message string
+		Type string
+	}
+}
+
+func (response CreateNamespaceResponse) ToError() error {
+	messages := []string{}
+
+	for i := range response.Errors {
+		messages = append(messages, response.Errors[i].Message)
+	}
+
+	return errors.New(strings.Join(messages, ": "))
+
+}
+
+func createNamespaceQuery(ctx context.Context, logger *logger.Logger, name string, ownerId string, response interface{}, query string) error {
```

Comment:
can you in-line this function into OrbCreateNamespace please? - I asked Eric to do the same at https://github.com/CircleCI-Public/circleci-cli/pull/27#discussion_r203738439 

Indeed if we merge Eric's PR and you re-base on top you can re-use his new Errors struct to remove `(response CreateNamespaceResponse) ToError()`

Reply:
✅